core/src/main/java/org/infinispan/persistence/file/SingleFileStore.java

```diff
@@ -106,13 +108,9 @@ public void init(InitializationContext ctx) {
 
    @Override
    public void start() {
+      Path location = PersistenceUtil.getQualifiedLocation(ctx.getGlobalConfiguration(), configuration.location(), ctx.getCache().getName() + ".dat");
```

Comment:
Should this include the `cacheName`?

Reply:
This was the format that we previously used for the actual file `<cache-name>.dat`. I just followed this convention.